plugin/src/sidebar.ts

```diff
@@ -114,9 +120,15 @@ reearth.on("update", () => {
   });
 });
 
-// reearth.on("resize", (e: any) => {
-//   if (e.isMobile !== isMobile) {
-//     reearth.ui.postMessage({type: ""});
-//     isMobile = e.isMobile;
-//   }
-// });
+reearth.on("resize", () => {
+  // if (e.isMobile !== isMobile) {
+  //   reearth.ui.postMessage({type: ""});
+  //   isMobile = e.isMobile;
+  // }
+
+  if (welcomPagIsOpen)
+    reearth.modal.update(welcomeScreenHtml, {
+      width: reearth.viewport.width,
+      height: reearth.viewport.height,
+    });
```

Comment:
Please recheck the API. I believe `reearth.modal.update` only takes an object as a param with width, height and background fields.

Reply:
done

Reply:
6dc6f5efc904b2988a326ea6674b939ecdca1183